packages/server/graphql/types/TemplateDimension.ts

```diff
@@ -70,12 +72,23 @@ const TemplateDimension = new GraphQLObjectType<any, GQLContext>({
       type: new GraphQLNonNull(GraphQLString),
       resolve: ({description}) => description || ''
     },
-    scale: {
+    selectedScale: {
       type: new GraphQLNonNull(TemplateScale),
       description: 'scale used in this dimension',
       resolve: ({scaleId}, _args, {dataLoader}) => {
         return dataLoader.get('templateScales').load(scaleId)
       }
+    },
+    availableScales: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(TemplateScale))),
+      description: 'The list of scales can be set for this dimension',
+      resolve: async ({teamId}, _args, {dataLoader}) => {
+        const activeTeamScales = await dataLoader.get('scalesByTeamId').load(teamId)
+        const publicScales = await db.read('starterScales', 'aGhostTeam')
+        const activeScales = [...activeTeamScales, ...publicScales]
+        activeScales.slice().sort((a, b) => (a.sortOrder < b.sortOrder ? -1 : 1))
```

Comment:
you'll want to remove `slice` here, otherwise you're copying an array, sorting it, and then returning the original.